firefox_privacy_notice/en-US.md

```diff
@@ -105,3 +105,8 @@ When you ask it to, Firefox also connects to Mozilla to provide you with feature
 {: #push-notifications }
 
 	Push Notifications allow sites to send notifications and updates to you if you opt-in. To receive notifications, Firefox sends information to Mozilla about what sites you have agreed to receive Push Notification from. We store this information in anonymized form, along with the number of notifications each site sends you. To help developers improve their use of Push Notifications, Mozilla may share aggregated information with certain developers including the number of visitors to their site that have subscribed or unsubscribed to their Push Notifications. You can manage Push Notifications in Firefox by following [these instructions](https://support.mozilla.org/kb/push-notifications-firefox).
+	
+## End "Things You Should Know"
```

Comment:
No, but it may be the incorrect way to address what I'm trying to indicate! The last bit of text (line 111) is intended to be OUTSIDE the "Things You Should Know" box, rather than INSIDE.